
try:
    raise NameError
except NameError as e:
    print(e)
